File: src/main/java/com/vyperplugin/psi/VyperTokenType.java
package com.vyperplugin.psi;

import com.intellij.psi.tree.IElementType;
import com.vyperplugin.Vyper;

public class VyperTokenType extends IElementType {

    public VyperTokenType(String debugName) {
        super(debugName, Vyper.INSTANCE);
    }

    @Override
    public String toString() {
        return "VyperTokenType." + super.toString();
    }
}

File: src/main/kotlin/com/vyperplugin/psi/VyperTokenType.kt
package com.vyperplugin.psi

import com.intellij.psi.tree.IElementType
import com.vyperplugin.Vyper

class VyperTokenType(debugName: String) : IElementType(debugName, Vyper.INSTANCE) {

    override fun toString(): String {
        return "VyperTokenType." + super.toString()
    }
}
